packages/types/src/Method.ts

```diff
@@ -66,6 +66,25 @@ export default class Method extends Struct {
    * - {@see DecodeMethodInput}
    * @param _meta - Metadata to use, so that `injectExtrinsics` lookup is not
    * necessary.
+   *
+   * @example
+   * <BR>
+   *
+   * ```javascript
+   * import { Method } from '@polkadot/types';
+   *
+   * const methodInstance = new Method(extrinsic, extrinsic.meta);
+   *
+   * // WRONG - never access `.raw` outside of the @polkadot/types package
```

Comment:
IMHO this should not go into the docs. The end-user docs should never mention `.raw`.

Reply:
@amaurymartiny I've pushed a commit to remove mention of `.raw` in user-facing docs